airbyte-integrations/connectors/source-mailgun/source_mailgun/spec.json

```diff
@@ -0,0 +1,16 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/mailgun",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Source Mailgun Spec",
+    "type": "object",
+    "required": ["private_key"],
```

Comment:
also for convenience it is better to use name like `api_key` or `api_token`, like in other places

Reply:
There are different kinds of API keys for this connector (which also could be implemented later), so its formal name fits the best.